src/app/search/state/search.service.ts

```diff
@@ -580,6 +581,35 @@ export class SearchService {
     ]);
   }
 
+  /**
+   * Initialize expanded panels to default state or restore previous state from local storage
+   */
+  initializeExpandedPanels() {
+    if (localStorage.getItem(this.expandedPanelsStorageKey)) {
+      return new Map<string, Boolean>(JSON.parse(localStorage.getItem(this.expandedPanelsStorageKey)));
+    } else {
+      return new Map([
```

Comment:
Looking at search.service.ts, we now have already had  4 maps where we initialize panels/buckets, now we have 5. It seems like they could be all combined into one data structure, but that would probably be a pretty massive change, so I don't know if it's worth it.

Reply:
Yeah, big change but we can probably make a ticket if this is something we want to do

Reply:
On further reflection, it doesn't have to be that big of a change. We could have one data structure, and derive all the maps from it, and not have to change any of the HTML. Please do create a ticket for it.